### ISSUE TITLE
AMD 17.6.1-June6 broke opentrack-PT tracker

### ISSUE THREAD

**Issue by albertolg** [opentrack/opentrack#635]
after I installed latest AMD drivers opentrack stopped working, PS3 eye, randomly stop working, but if I open the Cl-eye-test then the camera is working and I can see the 3 dots. Reverted back to AMD 17.5.2 and opentrack is working again... both versions 2.3.6 and 2.3.7 have the issue... hope AMD will fix with next release or I can't update the drivers. 

Windows 10 x64 1703

**Comment by sthalik** [comment]
@albertolg 

Download `AmCap` from http://opentrack.misaki.pl and tell me if that works.

**Comment by albertolg** [comment]
uummm... what should I see? I have 2 windows, a white one with a menu and nothing, the black one with numbers...

**Comment by sthalik** [comment]
The timestamp increments and buffer size is more than zero, yes?

This is the WIP camera support code but I haven't removed the entire user interface yet.

**Comment by albertolg** [comment]
yes, more than zero.. I'll try it again when my opentrack will froze... the problem was  the tracking frozing, than if I hit stop and start again it works again but randomly the problem occurs again... I will try reinstalling the new drivers, test and report back

**Comment by sthalik** [comment]
What if you plugged the webcam into another USB slot? That makes the USB device's drivers install again.

**Comment by albertolg** [comment]
haven't tried it yet, but what I noticed is that wen the problem happened I looked in opentrack and the dots in the monitor were stuck, but if I opened the CL-eye-test then the camera was working fine there but in opentrack was still stuck... don't you think if it was a problem with the drivers it could happen in every software I open? Reinstalled the new drivers, today I'll do more tests and if I can some videos...

**Comment by albertolg** [comment]
update: tested for about 2 hours and not a single problem... I don't know what happened before... well, happy is working again

**Comment by sthalik** [comment]
Your USB hu may be overloaded. Switch black/blue.

**Comment by albertolg** [comment]
no hub... but as I said the camera was still working... could have been a problem of not rebooting after new drivers, I don't know, I just uninstalled and installed the older then did a reboot.. today I used DDU

**Comment by sthalik** [comment]
What's DDU? Are you saying it works for you now?

**Comment by albertolg** [comment]
DDU= display driver uninstaller, an utility to clean the system from old graphic drivers. Yes, now it's working fine, I don't know what happened, was so strange see the tracking freezing, this never happened to me all this time. Imagine a paused old video tape, where the scene jump from a frame ahead and one back in a loop, well, this is what I have seen when this happened. something like this but without the distortion https://www.youtube.com/watch?v=917isMQyvjw

**Comment by albertolg** [comment]
just a little update.... the problems appeared again, also on 17.6.2, now I reverded back to 17.5.2 and will do more tests... this is really annoying

**Comment by albertolg** [comment]
another update... the issue is also with 17.5.2, damn it....
@sthalik is possible to activate a log so I can see what is happening in my system?

**Comment by sthalik** [comment]
@albertolg It's not the drivers. Switch the USB hub.

**Comment by albertolg** [comment]
@sthalik probably win10 failed to install the drivers the first time (I had win8.1 then updated to w10), because I didn't had the icon to remove the hardware in the traybar (i have a ps3 eye), now I uninstalled and then reinstalled the drivers, changed USB port and I have again the icon. Tested for a couple of hours and didn't had any issue. Probably the cam where disconnected randomly causing to opentrack the problem.
